modular_citadel/code/game/objects/items/devices/radio/shockcollar.dm

```diff
@@ -0,0 +1,83 @@
+/obj/item/electropack/shockcollar
```

Comment:
fetish content. in all seriousness though why is this in modular_citadel, and why is the diff showing it as new if it was already there

Reply:
how would i know lmao i don't wanna touch it haha